tests/unit/components/form/HdForm.spec.js

```diff
@@ -0,0 +1,194 @@
+import { wrapperFactoryBuilder } from 'tests/unit/helpers';
```

Comment:
Hey @ilyasmez and @kuroski, wouldn't it be valuable to test the dynamic input component? I think it is quite a critical path, right?

Reply:
I'll check on that, thanks for the comment man 😆

Reply:
Hey @viniciuskneves, what do you mean by "dynamic input component"? Do you mean HdDynamicForm? Or you're referring to the form components that use the new mixin?

Reply:
Hey @ilyasmez, I meant this: https://github.com/homeday-de/homeday-blocks/pull/588/files#diff-54b6a0b0dcd5da18fce5ff464cfad258f3699fba12d140ef276968d5865f17ffR84

Reply:
Ah, alright, got you, that makes sense.

Reply:
b95d87f021ca58ac69398e5cf6a76091f4116d71

Thanks for waiting, there we go @viniciuskneves